hoomd/simulation.py

```diff
@@ -141,6 +141,37 @@ def verbose_run(self, value):
         self._verbose = bool(value)
         self._cpp_sys.enableQuietRun(not self.verbose_run)
 
+    @property
+    def always_compute_pressure(self):
+        """Always compute the virial and pressure.
+
+        By default, HOOMD only computes the virial and pressure on timesteps
+        where it is needed (when :py:class:`hoomd.dump.GSD` writes
+        log data to a file or when using an NPT integrator). Set
+        ``always_compute_pressure`` to True to make the per particle virial,
+        net virial, and system pressure available to query any time by property
+        or through the :py:class:`hoomd.Logger` interface.
+
+        Enabling this flag will result in a moderate performance penalty
```

Comment:
This may look better in the documentation as a note since that will get special formatting, and draw the eye.

Reply:
Done.